services/path.py

```diff
@@ -165,6 +171,19 @@ def __init__(
                 if search_term.lower() in path.relative_path.lower()
             ]
 
+    @property
+    def files_accounting_flags(self) -> List[str]:
+        if not self.filter_flags:
+            return self.report.files
+        files = report_service.files_belonging_to_flags(
+            commit_report=self.report, flags=self.filter_flags
+        )
+        self._filter_commit_report()
```

Comment:
Not sure what I think about this, thought?

Reply:
I would maybe just do this in the constructor so we don't have to rely on this method being called.

```python
self.report = report
if self.flags_filter:
    self.report = self.report.filter(flags=self.flags_filter)
```

Reply:
I considered this, but the issue is that all the logic in the `files_accounting_flags` relies on a `Report` and not a `FilteredReport`. I suppose what I need to do is change that logic to use internally self.report(which is a `FitleredReport` by that point).report_file instead. I'll play around that

Reply:
Oh yeah, good point.  It's kinda strange to me that `Report` and `FilteredReport` don't offer the exact same API.